docs/README.md

```diff
@@ -10,7 +10,7 @@ All JavaScript is marked up with [JSDoc](https://github.com/jsdoc3/jsdoc) commen
 
 ## API documentation
 
-The API is documented via Swagger, but right now it’s only available when the site is running in Docker. That can be reviewed at [`http://localhost:3000/v1/swagger`](http://localhost:3000/v1/swagger).
+The API is documented via Swagger, but right now it’s only available when the site is running in Docker. That can be reviewed at [`http://localhost:3000/api/swagger`](http://localhost:3000/api/swagger).
```

Comment:
I have not touched this. I did look into it a few days ago and couldn't work out why it would be installing the older version of the dependency when cypress explicitly looks for `4.17.14`. I can give it another pass and see if something shakes out.

Reply:
`package-lock.json` has been updated to use `4.17.14` everywhere.